NET-943:  Fallback proxies (#1073)
streamr-dev/network: packages/client/README.md

```js

// Open publish proxy to multiple nodes on stream

> await publishingClient.setProxies(stream, ['0x11111...', '0x22222...'], ProxyDirection.PUBLISH)

Reviewer: The method doesn't handle a stream, but a stream part. We could highlight that fact in the readme, and  also rename the variable.

(One alternative would be to have the first parameter as `StreamDefinition[]` so that we could set proxy for the whole stream easily.)

```js

// Open publish proxy to multiple nodes on stream
await publishingClient.setProxies(stream, ['0x11111...', '0x22222...'], ProxyDirection.PUBLISH)

// Remove publish proxy to multiple nodes on stream

> await publishingClient.setProxies(stream, [], ProxyDirection.PUBLISH)

Reviewer: Could the comment or description be something like: "To use nodes "0x11111 and 0x22222 as proxies when subscribing to `stream`"?

And the comment or description about remove something like "To remove some/all proxies, call the same method with a different set of nodes. If the node list is empty, proxies are no longer used for the given stream."

```

> By default the client will attempt to open proxy connections to all of the nodes set in  `setProxies`. If you want to limit the number of connections by setting a fourth parameter. In this approach, if the client is disconnected from one of the nodes it will attempt to connect to another node by random.

Reviewer: ```suggestion
By default the client will attempt to open proxy connections to all of the nodes set in  `setProxies`. You can limit the number of connections by setting the `connectionCount` parameter. In this approach, if the client is disconnected from one of the nodes it will attempt to connect to another node by random.
```
